packages/flutter/lib/src/widgets/app.dart

```diff
@@ -197,6 +198,7 @@ class WidgetsApp extends StatefulWidget {
     this.shortcuts,
     this.actions,
     this.restorationScopeId,
+    this.scrollBehavior,
```

Comment:
(if we keep this, MaterialApp and CuppertinoApp should pass this in instead of wrapping with their own ScrollConfiguraiton)

Reply:
I go back and forth, I settled on yes since every app has to have a ScrollBehavior.. but I could go either way. I'll clean this up to have Material/Cupertino pass it and see how it looks.

Reply:
Actually, now that I play with it the other way, I think it's cleaner to go with the other option.